hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/compact/CompactionTriggerStrategy.java

```diff
@@ -18,15 +18,31 @@
 
 package org.apache.hudi.table.action.compact;
 
+import org.apache.hudi.common.config.EnumDefault;
+import org.apache.hudi.common.config.EnumDescription;
+import org.apache.hudi.common.config.EnumFieldDescription;
+
+@EnumDescription("Controls when compaction is scheduled")
 public enum CompactionTriggerStrategy {
```

Comment:
I just put the comments into these annotations. I'm pretty confused by the differences between NUM_COMMITS and NUM_COMMITS_AFTER_LAST_REQUEST especially. I can fix the wording after we figure out what is going on here.